wire-ios-data-model/Source/ManagedObjectContext/Migration/CoreDataMessagingMigrator.swift

```diff
@@ -60,9 +68,11 @@ extension CoreDataMessagingMigratorError: LocalizedError {
     }
 }
 
-final class CoreDataMessagingMigrator: CoreDataMessagingMigratorProtocol {
+extension CoreDataMessagingMigrator<CoreDataMessagingMigrationVersion>: CoreDataMessagingMigratorProtocol {}
```

Comment:
Why do we still have `CoreDataMessagingMigratorProtocol`?

Reply:
we need it for importBackup tests, I kept it as it is easier to keep the conformance and be able to inject a mock in the tests for backup for now